### PR TITLE
Routes and filters

### PR DESCRIPTION
**Pull request by khrystuk**

### CODE REVIEW

File: src/containers/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -15,26 +15,59 @@ class Dashboard extends Component {
...
+        this.setFilterTime = this.setFilterTime.bind(this);
+        this.fetchInvocationsData = this.fetchInvocationsData.bind(this);
     }
 
     componentWillMount() {
+        this.fetchInvocationsData(this.props.match.params.dashboardType);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (this.props.match.params.dashboardType !== nextProps.match.params.dashboardType) {
```
Comment (mrmckeb) [inline src/containers/Dashboard/Dashboard.js]: You may want to set these as variables for readability.

```
const currentDashboardType = ...;
const nextDashboardType = ...;
if (currentDashboardType !== nextDashboardType)
this.fetchInvocationsData(nextDashboardType);
```

File: src/containers/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -15,26 +15,59 @@ class Dashboard extends Component {
...
         }
     }
 
-    loadMoreHandler() {
+    setFilterTime(params) {
+        let time = new Date();
+        switch (params) {
+            case "week":
+                time.setDate(time.getDate() - 7);
+                return time.getTime();
```
Comment (mrmckeb) [inline src/containers/Dashboard/Dashboard.js]: Could you just always return `time.getTime()` after the switch?

File: src/containers/Dashboard/Dashboard.js (hunk 1 above)
Comment (vdanyliv) [reply src/containers/Dashboard/Dashboard.js]: we talked today about similar issue on other pr, you should think how to write your code more readable and simpler

File: src/containers/Dashboard/Dashboard.js (hunk 2 above)
Comment (khrystuk) [reply src/containers/Dashboard/Dashboard.js]: done

File: src/containers/Dashboard/Dashboard.js (hunk 1 above)
Comment (khrystuk) [reply src/containers/Dashboard/Dashboard.js]: fixed

**Pull request merged**